kcidb/monitor/output.py

```diff
@@ -136,9 +138,24 @@ def render(self):
             ready to send, but missing the From header.
         """
         email = EmailMessage()
-        email["Subject"] = self.message.summary + self.obj.summarize()
+        subject = self.message.summary + self.obj.summarize()
+        email["Subject"] = subject
         email["To"] = ", ".join(self.message.recipients)
         email["X-KCIDB-Notification-ID"] = self.id
         email["X-KCIDB-Notification-Message-ID"] = self.message.id
-        email.set_content(self.message.description + self.obj.describe())
+        message = self.message.description + self.obj.describe()
+        email.set_content(message)
+        formatted_subject = html.escape(subject)
+        formatted_message = html.escape(message)
```

Comment:
We shouldn't call these `formatted`, as that would be confusing (formatted usually means we used some sort of template to produce it), but instead call it e.g. `escaped`. Could you change that?

Reply:
Okay